ea-2022-04-ndvi-automation.ipynb

```diff
@@ -0,0 +1,1600 @@
+      "text/plain": [
+       "<Figure size 864x864 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Add only the plot code to this cell\n",
+    "\n",
+    "# This is the final figure of mean NDVI\n",
+    "# for both sites across the year\n",
+    "# with data cleaned to deal with clouds\n",
+    "\n",
+    "# Plot average NDVI values for both sites\n",
+    "fig, (ax) = plt.subplots(figsize=(12, 12))\n",
+    "\n",
+    "for site_name, group in mean_ndvi_df_total.groupby('site'):\n",
+    "    group.groupby('site').plot(y='mean_ndvi',\n",
+    "                               label=site_name,\n",
+    "                               linewidth=3.0,\n",
+    "                               ax=ax,\n",
+    "                               alpha=.8)\n",
+    "\n",
+    "# Set plot title and axes labels\n",
+    "ax.set(title=\"Mean NDVI Values from Landsat Data for Two Sites (Clouds Removed)\\n \\\n",
```

Comment:
While by no means necessary, consider adding the (fontweight='bold') parameter to the figure subtitles for a more aesthetic plot. Again this is by no means a requirement, just a personal preference, so feel free to consider it in the future!

Reply:
Yes - Thank you! I definitely will use in future notebooks, it helps the overall plot formatting a lot.